yb_migrate/src/utils/diagnostics.go

```diff
@@ -0,0 +1,143 @@
+package utils
+
+import (
+	"bytes"
+	"encoding/json"
+	"io/fs"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"path/filepath"
+	"time"
+
+	"github.com/google/uuid"
+	log "github.com/sirupsen/logrus"
+)
+
+//call-home json formats
+var (
+	jsonFilePath    string
+	sendReq         sendRequest
+	jsonBuf         []byte
+	SendDiagnostics bool
+)
+
+type sendRequest struct {
```

Comment:
consider naming as `Payload` since this is a datatype not a functionality.

Reply:
makes sense, done